src/Type/SchemaConfig.php

```diff
@@ -216,13 +216,13 @@ public function getTypes()
     }
 
     /**
-     * @param Type[]|callable $types
+     * @param array<Type>|(callable(): array<Type>) $types
```

Comment:
Missing array key type perhaps

Reply:
It is the same type we are discussing in https://github.com/webonyx/graphql-php/pull/770#discussion_r602849388